### ISSUE TITLE
Can't define() new properties on individual DefineMap instances.

### ISSUE THREAD

**Issue by bmomberger-bitovi** [canjs/can-define#83]
https://jsbin.com/vibazomace/edit?html,js,output

Success path:
- Make a constructor
- Define properties on its prototype with define()
- Create a new instance.
- Define new properties on the instance with define()
- Everything works.

Fail path:
- extend DefineMap to a new class with properties defined
- Create a new instance
- Define new properties on the instance with define()
- Results in an error: `Cannot redefine property: _data`

Is there any reason why this shouldn't work?

**Comment by justinbmeyer** [comment]
@bmomberger-bitovi I'm confused about what this issue is about.

On first glance, it looked like the issue is about creating a map like you did in the jsbin like:

```js
new can.DefineMap({
  constructor: function() {},
  _cid: Math.random().toString(10),
  _bindings: Math.random(),
  __bindEvents: {}
});
```

But the title and text of the issue makes me think it's about something different.  What do you mean by:

> Define new properties on the instance with define()

I don't see that happening in the jsbin.  Thanks!

**Comment by bmomberger-bitovi** [comment]
Are you looking at the right JSBin?  The code in question looks like this:

```javascript
  var baz = new Bar();
  
try {
    define(baz, {
      quux: { value: "jeek" }
    }, baz._define);
    document.body.innerHTML += new Bar().baz + " " + new Bar().quux;

  } catch(e) {
    document.body.innerHTML += "<span style='color:red'>" + e.message + "</span>";    
  }
```

That's trying to define on `baz`, which is type `Bar`  (note that it's _not_ being done on Bar's prototype but on a created instance)

Also the latest steal breaks when setting `config: "@empty"` so here's a working one: https://jsbin.com/qoyegutayu/edit?html,js,output

**Comment by bmomberger-bitovi** [comment]
I think you were looking at the JSBin for #92

**Comment by bmomberger-bitovi** [comment]
JSBin updated to reflect some new dependencies from can-util: https://jsbin.com/vibazomace/edit?html,js,output

**Comment by bmomberger-bitovi** [comment]
Closed via #189
